pkg/services/sqlstore/quota.go

```diff
@@ -98,15 +99,17 @@ func UpdateOrgQuota(cmd *m.UpdateOrgQuotaCmd) error {
 	return inTransaction(func(sess *DBSession) error {
 		//Check if quota is already defined in the DB
 		quota := m.Quota{
-			Target: cmd.Target,
-			OrgId:  cmd.OrgId,
+			Target:  cmd.Target,
```

Comment:
Are these changes related to the rest of the PR? seems odd :)

Reply:
As explained in the commit message there is 2 side effects. One of them is that zero time is now translated to nil before sql insertion. This raise errors in tests that were not previously raised: `'NOT NULL constraint failed: dashboard_acl.created'`.
From my point of view, error are rightly raised since created/updated fields are sometimes never filled.
So I filled such (missing ?) fields for `quota` and `dashboard_acl`.